persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -115,6 +123,9 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
       log.debug(s"Parsing completed for $submissionId")
       publishEvent(ParsingCompleted(submissionId))
 
+    case CompleteParsingWithErrors =>
+      log.debug(s"Parsing completed for $submissionId, errors found")
```

Comment:
See what I wrote a bit about this in the original issue, #552. I'm sending the event `ParsingCompletedWithErrors`, because that follows the pattern that the `HmdaFileValidator` established [here](https://github.com/cfpb/hmda-platform/blob/master/persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala#L134).

If the event should really be called `ParsedWithErrors`, should I change the `HmdaFileValidator` to follow the same pattern?